packages/app/src/app/pages/Dashboard/Content/DragLayer/index.js

```diff
@@ -37,8 +35,6 @@ class CustomDragLayer extends React.Component {
         isOverPossibleTargets={isOverPossibleTargets}
         x={x}
         y={y}
-        left={left}
-        top={top}
```

Comment:
Why are you not passing this one down anymore?

Reply:
The reason I removed these `props` is because they are not used any where by any components under `<SelectedSandboxItems />`.
You can see they are forwarded in [here](https://github.com/codesandbox/codesandbox-client/blob/master/packages/app/src/app/pages/Dashboard/Content/DragLayer/SelectedSandboxItems/index.js#L29) to `<AnimatedSandboxItem />`, but `<AnimatedSandboxItem />` does not use them and not expects them either [here](https://github.com/codesandbox/codesandbox-client/blob/master/packages/app/src/app/pages/Dashboard/Content/DragLayer/SelectedSandboxItems/AnimatedSandboxItem.js#L11).
It might be a bug for `<AnimatedSandboxItem />` not using them, then we can fix it with more context knowledge.